source/ClientDebug/ClientDebug.csproj

```
@@ -43,17 +44,13 @@
     <RunPostBuildEvent>Always</RunPostBuildEvent>
   </PropertyGroup>
   <ItemGroup>
-    <None Include="App.config" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Main.cs" />
+    <PackageReference Include="Bannerlord.Lib.Harmony" Version="2.2.2" />
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
     <PreBuildEvent>
     </PreBuildEvent>
   </PropertyGroup>
   <PropertyGroup>
-    <PostBuildEvent>"$(DevEnvDir)devenv" "$(SolutionDir)Coop.sln" /Rebuild $(configuration) /project "$(SolutionDir)Coop\Coop.csproj"</PostBuildEvent>
+    <PostBuildEvent>dotnet build $(SolutionDir)Coop/Coop.csproj --no-incremental -c $(configuration) /p:SolutionDir=$(SolutionDir)</PostBuildEvent>
```

This requires quotes on windows otherwise

```Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB1008	Only one project can be specified.	ClientDebug	D:\Mod Development\BannerlordCoop\source\ClientDebug\MSBUILD	1	
```
```suggestion
    <PostBuildEvent>dotnet build "$(SolutionDir)Coop/Coop.csproj" --no-incremental -c $(configuration) /p:"SolutionDir=$(SolutionDir)/"</PostBuildEvent>
```

---

Done